data_ingestion.ipynb

```diff
@@ -22,7 +22,7 @@
         "colab_type": "text"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    dataset_root = OUTPUT_DATASET_ROOT
if SHARED_DRIVE:
&nbsp; dataset_root = os.path.join(
&nbsp; &nbsp; &nbsp; "/content/gdrive/Shareddrives/TNC Fellowship 🌳/4. Isotope Research &amp; Signals/code/amazon_rainforest_files/amazon_sample_data",
&nbsp; &nbsp; &nbsp; dataset_root
&nbsp; )
else:
&nbsp; dataset_root = raster.get_sample_db_path(dataset_root)
</pre><p> </p><p>Is a notebook the right place for this logic? More a matter of curiosity than something to address.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/197/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Yeah, since we take an input from the checkbox and the logic that's complex has already been abstracted i.e. get_sample_db_path.